src/main/resources/static/js/identifyHandler.js

```diff
@@ -1,5 +1,6 @@
+const hash = new URL(window.location).searchParams.get("hash");
```

Comment:
How about renaming this file to something like `setEIDClientURLToStartIdentButton`? Or maybe only `setEIDClientURL`?

Reply:
With pleasure. 👍 Don't like the current name, either.